pkg/cloud/services/networking/securitygroups.go

```diff
@@ -141,8 +141,8 @@ func generateControlPlaneGroup(clusterName string) infrav1.SecurityGroup {
 				{
 					Direction:      "ingress",
 					EtherType:      "IPv4",
-					PortRangeMin:   443,
-					PortRangeMax:   443,
+					PortRangeMin:   6443,
+					PortRangeMax:   6443,
```

Comment:
@hidekazuna do we already have rules for port 22? Just because when we want to add a e2e test for Octavia we have to be able to jump via port 22 onto the machines

Reply:
P.S. we can add a Octavia test in the future, but I would like to stabilize the current test first.

Reply:
@sbueringer Yes, we have. See line 149-155.  Opening port 22 from all is limited to master nodes. Worker nodes are accessed from only master nodes. If worker nodes have to be accessed from externally, we need to fix.

Current security group rules are the followings.

```bash
secgroup-controlplane
+--------------------------------------+-------------+-----------+------------+-----------+-----------+-----------------------+
| ID                                   | IP Protocol | IP Range  | Port Range | Direction | Ethertype | Remote Security Group |
+--------------------------------------+-------------+-----------+------------+-----------+-----------+-----------------------+
| 10f01e25-567b-42cf-bbf6-9a7a52e63cca | None        | None      |            | egress    | IPv4      | None                  |
| e3f0adfb-2fd5-4e11-8bba-153434b9e959 | None        | None      |            | egress    | IPv6      | None                  |
| 41b90c1b-2afe-48ca-9866-c8fd60c6d639 | tcp         | 0.0.0.0/0 | 22:22      | ingress   | IPv4      | None                  |
| b9e90307-088e-44c6-838e-b3d1305dd85a | tcp         | 0.0.0.0/0 | 6443:6443  | ingress   | IPv4      | None                  |
+--------------------------------------+-------------+-----------+------------+-----------+-----------+-----------------------+

secgroup-all
+--------------------------------------+-------------+----------+------------+-----------+-----------+--------------------------------------+
| ID                                   | IP Protocol | IP Range | Port Range | Direction | Ethertype | Remote Security Group                |
+--------------------------------------+-------------+----------+------------+-----------+-----------+--------------------------------------+
| 64b801bc-4523-485f-a480-7ff4f0e26ad1 | None        | None     |            | egress    | IPv4      | None                                 |
| 9687d4f1-23bf-478e-b628-86a62da6b91b | None        | None     |            | egress    | IPv6      | None                                 |
| 631b72b1-b895-4a5f-b953-f1febb3c8979 | udp         | None     | 1:65535    | ingress   | IPv4      | 906e8b6e-e55b-40f8-a62e-4f8bb5da4e6d |
| 66a91b6a-6d49-457e-9a99-eebc8e234f7a | tcp         | None     | 1:65535    | ingress   | IPv4      | 906e8b6e-e55b-40f8-a62e-4f8bb5da4e6d |
| 736d7dab-5e9e-4d81-baf7-8fe4ad7745ab | icmp        | None     |            | ingress   | IPv4      | 906e8b6e-e55b-40f8-a62e-4f8bb5da4e6d |
+--------------------------------------+-------------+----------+------------+-----------+-----------+--------------------------------------+
```

Reply:
If I got it correctly we can jump to master nodes and from master to worker? Then it's fine

Reply:
Yes, You are right.